### ISSUE TITLE
Alert when refreshing the page with the browser if a form is not empty and not submitted

### ISSUE THREAD

**Issue by marc-farre** [humhub/humhub#6708]
### Is your feature request related to a problem? Please describe.

Original discussion: https://github.com/humhub/app/issues/147

1. Fill a form and don't submit it.
2. Refresh the browser page.
3. The page refreshes and in some cases the content entered in the form is lost (for the "What's in your mind" form field, the text is recovered)

### Describe the solution you'd like

When an internal link is clicked, we have a warning dialog: "Unsaved changes will be lost. Do you want to proceed?".
https://github.com/humhub/humhub/blob/master/protected/humhub/widgets/CoreJsConfig.php#L50

It should also ask us to confirm when refreshing a page.

**Comment by marc-farre** [comment]
Issue for the Messenger module: https://github.com/humhub/mail/issues/364

**Comment by yurabakhtin** [comment]
@marc-farre I have detected the confirmation message is not displayed on refresh a page only for this case:

- load a page with RichText editor (e.g. the post create form with field "What's in your mind")
- type some text there, so the RichText is focused, press F5 or Ctrl+R
- at this time a value of `<textarea>` of the RichText field is not updated, because it is udpated only on unfocus event
- so my fix was to unfocus all form's elements before unload a page.

Fixed in PR https://github.com/humhub/humhub/pull/6736.
